eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxow.hs

```diff
@@ -264,10 +265,11 @@ alonzoStyleWitness = do
 
   {-  { h | (_ → (a,_,h)) ∈ txins tx ◁ utxo, isNonNativeScriptAddress tx a} = dom(txdats txw)   -}
   let inputs = getField @"inputs" txbody :: (Set (TxIn (Crypto era)))
-      smallUtxo = eval (inputs ◁ utxo) :: Map.Map (TxIn (Crypto era)) (Core.TxOut era)
+      smallUtxo :: [(TxIn (Crypto era), Core.TxOut era)]
+      smallUtxo = SplitMap.toList (unUTxO utxo `SplitMap.restrictKeysSet` inputs)
       twoPhaseOuts =
```

Comment:
Perhaps like this?

smallUtxo = SplitMap.toList (inputs Split.◁ utxo)

Reply:
Done